src/Microsoft.Diagnostics.Runtime/src/DacInterface/ClrDataProcess.cs

```diff
@@ -118,13 +109,12 @@ public IEnumerable<ClrDataMethod> EnumerateMethodInstancesByAddress(ClrDataAddre
 
             List<ClrDataMethod> result = new List<ClrDataMethod>(1);
 
-            int hr = _startEnum(Self, addr, IntPtr.Zero, out ClrDataAddress handle);
-            if (hr != S_OK)
+            if (!_startEnum(Self, addr, IntPtr.Zero, out ClrDataAddress handle))
                 return result;
 
             try
             {
-                while ((hr = _enum(Self, ref handle, out IntPtr method)) == S_OK)
+                while (_enum(Self, ref handle, out IntPtr method))
```

Comment:
HResult.S_OK is zero, this change breaks this function.

Reply:
Anything wrong?

#### Before

`hr == 0`

#### After

`hr >= 0`

Reply:
Yep that's a bug.  Thanks!  I've fixed this here, which will be released in a nuget package this week:  https://github.com/microsoft/clrmd/pull/819